stable/influxdb/templates/post-install-set-auth.yaml

```diff
@@ -20,7 +20,7 @@ spec:
     spec:
       containers:
       - name: {{ template "influxdb.fullname" . }}-set-auth
-        image: "{{ .Values.setDefaultUser.image }}"
+        image: "{{ default "appropriate/curl:latest" .Values.setDefaultUser.image }}"
```

Comment:
Uncomment the defaults in `values.yaml` instead of using the `default` function.